polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,371 @@
+
+According to Polkadot's design, any chain that can compile to WebAssembly (Wasm) and complies with the  chain's API can integrate into the Polkadot network as a parachain.
+
+Here’s a high-level overview of the Polkadot protocol architecture
+
+![](/images/polkadot-protocol/architecture/polkadot-chain/overview/relay-chain-architecture.webp)
+
+Polkadot chains propose blocks to the Polkadot validators, who perform checks for availability and validity before finalizing them. With the  chain providing security, collators — full nodes of Polkadot chains — can focus on their tasks without needing strong incentives.
+
+The [Cross-Consensus Messaging Format (XCM)](add link to XCM page){target=\_blank} allows Polkadot chains to exchange messages freely, leveraging the  chain's security for trust-free communication.
+
+In order to interact with chains that want to use their own finalization process (e.g. Bitcoin), Polkadot has [bridges](add link to bridges pages, Snowbridge, etc){target=\_blank}  that offer two-way compatibility, meaning that transactions can be made between different Polkadot chains.
+
+### Polkadot's Additional Functionalities
+
+The Polkadot  chain oversees crowdloans and auctions. Chain cores were leased through auctions for three-month periods, up to a maximum of two years. 
+
+Crowdloans enabled users to securely lend funds to teams for lease deposits in exchange for pre-sale tokens, which is the only way to access slots on Polkadot 1.0.
+
+!!!note
+    Auction are deprecated in favor of Coretime
+
+Additionally, the  chain handles [staking](add link to PoS page){target=\_blank}, [accounts](add a link to accounts pages){target=\_blank}, balances, and [governance](add a link to govenance page){target=\_blank}.
+
+### Polkadot's Resilience
+
+Decentralization is a vital component of blockchain networks, but it comes with trade-offs:
+
+- An overly decentralized network may face challenges in reaching consensus and require significant energy to operate.
+- Also a network that achieves consensus quickly risks centralization, making it easier to manipulate or attack.
+
+Ideally, a network should be decentralized enough to prevent any manipulative or malicious influence. In this sense, decentralization serves as a tool to achieve resilience. 
+
+**Polkadot 1.0** currently achieves resilience through several strategies:
+
+1. **Nominated Proof of Staking (NPoS)**: This ensures that the stake per validator is maximized and evenly distributed among validators.
+
+2. **1KV Program**: Designed to encourage new operators to join the network, this program enhances both physical decentralization (the number of validator nodes per service provider) and social decentralization (the number of validator nodes per operator). Feel free to explore these metrics using the [Polkawatch App](https://polkadot.polkawatch.app/){target=\_blank}.
```

Comment:
This program is being sunset. The new one is [Decentralized Nodes](https://nodes.web3.foundation/)

Reply:
updated